proto/observer/crosschain_flags.proto

```diff
@@ -0,0 +1,22 @@
+syntax = "proto3";
+package zetachain.zetacore.observer;
+
+import "gogoproto/gogo.proto";
+import "google/protobuf/duration.proto";
+
+option go_package = "github.com/zeta-chain/zetacore/x/observer/types";
+
+message GasPriceIncreaseFlags {
+  int64 epochLength = 1;
```

Comment:
do we allow negative epochLength (e.g., special meaning)?

Reply:
3553a17a